base/markdown/GitHub/GitHub.jl

```diff
@@ -33,7 +33,83 @@ function github_paragraph(stream::IO, md::MD, config::Config)
     return true
 end
 
-# TODO: tables
+typealias Row Vector
 
-@flavor github [list, indentcode, blockquote, fencedcode, hashheader, github_paragraph,
-                 en_dash, inline_code, asterisk_bold, asterisk_italic, image, link]
+type Table
+    rows::Vector{Row}
+    align
+end
+
+function github_table(stream::IO, md::MD, config::Config)
+    withstream(stream) do
+        rows = Row[]
+        n = 0
+        align = :r # default is to align right
+        while !eof(stream)
+            n += 1
+            pos = position(stream)
```

Comment:
Can we refactor this part once we have some tests, after this is merged? (And we have some tests). I think I've addressed your other points in: https://github.com/hayd/julia/commit/ee27d425bfc56e15abd2a31e9b24adb204a97693 Thanks!